workspaces/libnpmpublish/lib/provenance.js

```diff
@@ -19,9 +19,10 @@ const generateProvenance = async (subject, opts) => {
   let payload
   if (ci.GITHUB_ACTIONS) {
     /* istanbul ignore next - not covering missing env var case */
-    const [workflowPath, workflowRef] = (env.GITHUB_WORKFLOW_REF || '')
+    const [workflowPath, ...rest] = (env.GITHUB_WORKFLOW_REF || '')
       .replace(env.GITHUB_REPOSITORY + '/', '')
       .split('@')
```

Comment:
Would an alternative be to split with the second argument as 1?

Reply:
In that case, the `rest` would be `undefined`

Reply:
Are you sure it wouldn’t just be the rest of the string?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split#limit
> Any leftover text is not included in the array at all.

```bash
$ node -e "console.log('a@b@c'.split('@', 1))"
[ 'a' ]
```